index.html

```diff
@@ -104,53 +105,79 @@ <h1>Plug and play</h1>
 </section>
 
 <section class="products-section">
-    <div class="product-presentation">
-        <h1 class="product-mission-header">On a mission?</h1>
-        <a href="/get-started-right/" class="product product-new-platform">
-            <img src="/img/map-icon.svg">
-            <h1>Put your business plan into action</h1>
-            <p>
-                If you have your business plan for your tech company ready, we can help you figure out how to make the right technical choices, and help you start your software development department the right way.
-            </p>
-            <p>
-                We work well within a budget, and provide the same value cheaper than any other option.
-            </p>
-            <div class="read-more">Get started right</div>
-        </a>
-        <a href="/start-automating/" class="product product-automation">
-            <img src="/img/input-icon.svg">
-            <h1>Automate away the tiresome tasks</h1>
-            <p>
-                Business is going well, but as your company has grown, more and more time is spent on drudge work.
-                We can help you identify and automate boring work, freeing up more time for your employees to focus on what is important.
-            </p>
-            <p>
-                We help ensure a constructive process for and buy-in from all stakeholders.
-            </p>
-            <div class="read-more">Start automating</div>
-        </a>
-        <h1 class="product-problem-header">Got a problem?</h1>
-        <a href="/get-awesome/" class="product product-fix-platform">
-            <img src="/img/layers-icon.svg">
-            <h1>Make your existing platform awesome!</h1>
-            <p>
-                If you already have a tech product we can help you maintain it, expand it with new features, and prepare it for growth.
-                We ensure that maintenance is efficient and stays within your budget.
+    <h1 class="title">What we excel at</h1>
```

Comment:
`.title` er nok lidt for generic en class, medmindre det er en class vi genbruger genrelt?

Reply:
Tekstforslag: "We excel at solving these problems..." (brug &hellip; til at lave "...")